src/hotspot/share/oops/method.hpp

```diff
@@ -820,78 +795,29 @@ class Method : public Metadata {
   void init_intrinsic_id(vmSymbolID klass_id);     // updates from _none if a match
   static vmSymbolID klass_id_for_intrinsics(const Klass* holder);
 
-  bool caller_sensitive() {
-    return (_flags & _caller_sensitive) != 0;
-  }
-  void set_caller_sensitive(bool x) {
-    _flags = x ? (_flags | _caller_sensitive) : (_flags & ~_caller_sensitive);
-  }
-
-  bool force_inline() {
-    return (_flags & _force_inline) != 0;
-  }
-  void set_force_inline(bool x) {
-    _flags = x ? (_flags | _force_inline) : (_flags & ~_force_inline);
-  }
-
-  bool dont_inline() {
-    return (_flags & _dont_inline) != 0;
-  }
-  void set_dont_inline(bool x) {
-    _flags = x ? (_flags | _dont_inline) : (_flags & ~_dont_inline);
-  }
-
-  bool changes_current_thread() {
-    return (_flags & _changes_current_thread) != 0;
-  }
-  void set_changes_current_thread(bool x) {
-    _flags = x ? (_flags | _changes_current_thread) : (_flags & ~_changes_current_thread);
-  }
+  bool caller_sensitive() const     { return constMethod()->caller_sensitive(); }
+  void set_caller_sensitive() { constMethod()->set_caller_sensitive(); }
 
-  bool jvmti_mount_transition() {
-    return (_flags & _jvmti_mount_transition) != 0;
-  }
-  void set_jvmti_mount_transition(bool x) {
-    _flags = x ? (_flags | _jvmti_mount_transition) : (_flags & ~_jvmti_mount_transition);
-  }
+  bool changes_current_thread() const { return constMethod()->changes_current_thread(); }
+  void set_changes_current_thread() { constMethod()->set_changes_current_thread(); }
 
-  bool is_hidden() const {
-    return (_flags & _hidden) != 0;
-  }
+  bool jvmti_mount_transition() const { return constMethod()->jvmti_mount_transition(); }
+  void set_jvmti_mount_transition() { constMethod()->set_jvmti_mount_transition(); }
 
-  void set_hidden(bool x) {
-    _flags = x ? (_flags | _hidden) : (_flags & ~_hidden);
-  }
+  bool is_hidden() const { return constMethod()->is_hidden(); }
+  void set_hidden() { constMethod()->set_is_hidden(); }
```

Comment:
The naming is inconsistent here regarding `is` - either both should have it or neither IMO.

Reply:
This one is easy to fix because there aren't calls everywhere.